db/upgrade.php

```diff
@@ -36,17 +36,17 @@
  * @param int $oldversion
  * @return bool
  */
-function xmldb_block_evasys_sync_upgrade($oldversion) {
+function xmldb_block_evasys_sync_upgrade ($oldversion) {
     global $DB;
     $dbman = $DB->get_manager();
-    if ($oldversion < 2017121403) {
+    if ($oldversion < 2019132600) {
 
```

Comment:
I think you should leave the $oldversion < 2017121403 branch as it is and create a new one below.

Reply:
I changed all three tables, therefore I'd like to have all of them in one place. I could take the definition of the first table out into a different branch, however I'd then need it in the new one anyway